LunaMary/src/IQuestion.java

```diff
@@ -0,0 +1,19 @@
+public interface IQuestion {
```

Comment:
Please make sure you need all these methods, if not remove the unnecessary ones.